data/mods/My_Sweet_Cataclysm/sweet_mapgen/sweet_overmap_locations.json

```diff
@@ -0,0 +1,335 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "pond_swamp_chocolate" ],
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+        "                        ",
+        "                        ",
+        "     ????????????????   ",
+        "   ???????????????????  ",
+        "   ?**]]]]]*]***]]]]??  ",
+        "  ???**************]??  ",
+        "  ???*************]]??  ",
+        "  ????***[[[*******]??  ",
+        "  ???***[[[[[[[[**]]??  ",
+        "  ???***[[[[[[[[**]]??  ",
+        "  ???***[[[~~~[[**]]??  ",
+        "  ??****[[[~~~[[***]??  ",
+        "  ??]***[[~~~~[[***]??  ",
+        "  ??]***[[~~~~[[**]]??  ",
+        "  ??]***[[~[[[[[**]]??  ",
+        "  ??]***[[[[[[[[**]]??  ",
+        "  ??]*****[[******]]??  ",
+        "  ??]**************]??  ",
+        "  ??]**************]??  ",
+        "  ????]??]]]]]]]]]]]??  ",
+        "  ????????]]]????????   ",
+        "  ?????????????????     ",
+        "                        ",
+        "                        "
+      ],
+      "//": "add furniture of hard rock candy replacing rubble_rock and boulders, maybe cattails.",
+      "set": [
+        { "point": "furniture", "id": "f_rubble_rock", "x": [ 1, 22 ], "y": [ 1, 22 ], "chance": 8, "repeat": [ 1, 5 ] },
+        {
+          "point": "furniture",
+          "id": "f_boulder_small",
+          "x": [ 1, 22 ],
+          "y": [ 1, 22 ],
+          "chance": 8,
+          "repeat": [ 1, 5 ]
+        },
+        {
+          "point": "furniture",
+          "id": "f_boulder_medium",
+          "x": [ 1, 22 ],
+          "y": [ 1, 22 ],
+          "chance": 8,
+          "repeat": [ 1, 5 ]
+        },
+        {
+          "point": "furniture",
+          "id": "f_boulder_large",
+          "x": [ 1, 22 ],
+          "y": [ 1, 22 ],
+          "chance": 8,
+          "repeat": [ 1, 5 ]
+        },
+        { "point": "furniture", "id": "f_cattails", "x": [ 1, 22 ], "y": [ 1, 22 ], "chance": 3, "repeat": [ 1, 5 ] },
+        { "point": "furniture", "id": "f_cattails", "x": [ 1, 22 ], "y": [ 1, 22 ], "chance": 3, "repeat": [ 1, 5 ] }
+      ],
+      "terrain": {
+        " ": [
+          "t_chocolate_dp",
+          "t_chocolate_dp",
+          "t_chocolate_sh",
+          "t_chocolate_sh",
+          "t_chocolate_sh",
+          "t_dirt",
+          "t_dirt",
+          "t_dirt",
+          "t_dirt",
+          "t_grass",
+          "t_grass",
+          "t_grass",
+          "t_grass",
+          "t_grass",
+          "t_grass",
+          "t_shrub",
+          "t_underbrush",
+          "t_tree_young",
+          "t_tree",
+          "t_tree_willow"
+        ],
```

Comment:
These terrain entries can all be condensed for easier reading/updating using this format:
" ": [ [ "t_chocolate_dp", 2 ], [ "t_chocolate_sh", 3 ], [ "t_dirt, 4 ], [ "t_grass", 6 ] "t_underbrush", "t_shrub", "t_tree_young", "t_tree", "t_tree_willow" ],